Passing -arch_only to libtool breaks building a PPC-only framework

--- Makefile.pre.arch_only	2008-09-28 21:45:54.000000000 -0400
+++ Makefile.pre	2008-09-28 21:47:07.000000000 -0400
@@ -395,7 +395,7 @@
 			-current_version $(VERSION); \
         else \
 		/usr/bin/libtool -o $(LDLIBRARY) -dynamic $(OTHER_LIBTOOL_OPT) $(LIBRARY) \
-			 -lSystem -lSystemStubs -arch_only i386 -install_name $(PYTHONFRAMEWORKINSTALLDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK) -compatibility_version $(VERSION) -current_version $(VERSION) ;\
+			 -lSystem -lSystemStubs -install_name $(PYTHONFRAMEWORKINSTALLDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK) -compatibility_version $(VERSION) -current_version $(VERSION) ;\
 	fi
 	$(INSTALL) -d -m $(DIRMODE)  \
 		$(PYTHONFRAMEWORKDIR)/Versions/$(VERSION)/Resources/English.lproj
